FreeRTOS-Plus/Source/FreeRTOS-Plus-TCP/FreeRTOS_ARP.c

```diff
@@ -256,52 +260,56 @@ uint8_t ucMinAgeFound = 0U;
 				if( memcmp( xARPCache[ x ].xMACAddress.ucBytes, pxMACAddress->ucBytes, sizeof( pxMACAddress->ucBytes ) ) == 0 )
 				{
 					/* This function will be called for each received packet
-					As this is by far the most common path the coding standard
-					is relaxed in this case and a return is permitted as an
-					optimisation. */
+					 * As this is by far the most common path the coding standard
+					 * is relaxed in this case and a return is permitted as an
+					 * optimisation. */
 					xARPCache[ x ].ucAge = ( uint8_t ) ipconfigMAX_ARP_AGE;
-					xARPCache[ x ].ucValid = ( uint8_t ) pdTRUE;
+					xARPCache[ x ].ucValid = ( uint8_t ) pdTRUE;					
 					return;
 				}
 
 				/* Found an entry containing ulIPAddress, but the MAC address
-				doesn't match.  Might be an entry with ucValid=pdFALSE, waiting
-				for an ARP reply.  Still want to see if there is match with the
-				given MAC address.ucBytes.  If found, either of the two entries
-				must be cleared. */
+				 * doesn't match.  Might be an entry with ucValid=pdFALSE, waiting
```

Comment:
These comments are inconsistent when representing equality; see the comment on line 254.  I'd suggest modifying this comment as `ucValid == pdFALSE`.

Reply:
done